### ISSUE TITLE
[Bug]: 1.19.2, Tempad 1.4.5 + Minetogether 6.2.4 crash on loading screen

### ISSUE THREAD

**Issue by Mundif** [terrarium-earth/Tempad#108]
### Bug Description

When having both mods installed with those versions (most recent), game crash on bar loading screen.
Loads fine with version 6.2.3 of Minetogether.

### How to Reproduce?

Used FTB app, created new instance, installed most recent Tempad and Minetogether, game crash at loading screen,
Downgraded Minetogether to 6.2.3 and game launches correctly.

### Version

1.4.5 for 1.19.2

### Mod Loader Version

1.19.2 - 43.3.9 Forge

### Mod Loader

Forge

### Code of Conduct

- [X] I have searched the issue tracker and confirmed that the issue has not been reported
- [X] I have checked the FAQ (if one is present) and my issue is not listed
- [X] I have verified that I am using the latest version of the mod
- [X] I have verified that I have installed all the required dependencies for the mod
- [X] I have verified that I do not have Optifine installed. We will close issues if we detect Optifine is in your pack. We cannot fix Optifine issues. Please do not report issues with Optifine present.

**Comment by CodexAdrian** [comment]
Please include a crash report

**Comment by Mundif** [comment]
[crash-2024-04-07_20.55.52-client.txt](https://github.com/terrarium-earth/Tempad/files/14898549/crash-2024-04-07_20.55.52-client.txt)

Here you go

**Comment by CodexAdrian** [comment]
You're telling me this *only* happens with the newer version of Minetogether? Does downgrading it again make the crash go away?

**Comment by Mundif** [comment]
Yes, I can upgrade downgrade infinite number of times, when I downgrade it works, when using most recent - crash

**Comment by CodexAdrian** [comment]
Fascinating. Thank you! This crash has been happening for a while but ive never had a way to replicate this crash before. I think i can finally fix this issue now!!!!!!

**Comment by Mundif** [comment]
Glad to help, happy coding :)
